### PR TITLE
welcome page

### LINKED ISSUE #13: Create cool welcome page
### Create welcome page for user in Client side

**_The idea is that this page should be the first that the user sees and from it he can go to the authorization page or register a new account_**

_Pls @Serhiy-dotcom @Varvara-Maria @Vova-Romaniuk I want to hear your opinion:)_

Something like this:
image

or this 
image

### PR DESCRIPTION
**Pull request by Serhiy-dotcom**

### CODE REVIEW

**Review by Vova-Romaniuk — APPROVED** [review]

File: StudCity.Frontend/src/components/Svg/index.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
```
Comment (Vova-Romaniuk) [inline StudCity.Frontend/src/components/Svg/index.jsx]: Please delete this empty string, in this place should have been import....
Fixed this problems in other files

**Review by Vova-Romaniuk — CHANGES_REQUESTED** [review]

File: StudCity.Frontend/.eslintrc.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
         "react"
     ],
     "rules": {
-        "react/prop-types": "off",
+		"react/prop-types": "off",
```
Comment (maryanchuk04) [inline StudCity.Frontend/.eslintrc.json:23]: fix this tab

File: StudCity.Frontend/src/pages/Main/index.jsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+						<NavItem to='register' className={transparentText}>Register</NavItem>
+					</div>
+					<Svg type="welcomeSvg" />
+				</div>
+			</div>
+		</div>
+	)
+}
+
+const NavItem = ({ children, to = '/', className = '' }) => {
```
Comment (maryanchuk04) [inline StudCity.Frontend/src/pages/Main/index.jsx]: Bro, you can take it to a separate component

**Review by maryanchuk04 — CHANGES_REQUESTED** [review]
Pls fix some comments

File: StudCity.Frontend/src/pages/Main/index.jsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
```
Comment (maria2908) [inline StudCity.Frontend/src/pages/Main/index.jsx]: fix this enter

Comment (Vova-Romaniuk) [reply StudCity.Frontend/src/pages/Main/index.jsx]: +  read the comment above

**Review by Vova-Romaniuk — CHANGES_REQUESTED** [review]

**Review by maria2908 — APPROVED** [review]
great job Sirganski :)

**Review by Vova-Romaniuk — APPROVED** [review]

**Review by maryanchuk04 — APPROVED** [review]
LGTM!

**Pull request merged**